http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/listener/WebSocketSourceHandler.java

```diff
@@ -144,13 +150,9 @@ protected void publishToMessageProcessor(CarbonMessage cMsg) {
         }
     }
 
-    /*
-    This method runs when initiating a connection.
-     */
-    private void sendOnOpenMessage(ChannelHandlerContext ctx, boolean isSecured, String uri) {
+    private void sendOnOpenMessage(ChannelHandlerContext ctx, boolean isSecured, String uri) throws URISyntaxException {
         cMsg = new StatusCarbonMessage(org.wso2.carbon.messaging.Constants.STATUS_OPEN, 0, null);
         Session session = new WebSocketSessionImpl(ctx, isSecured, uri, channelId);
-        WebSocketSessionManager.getInstance().add(uri, session);
```

Comment:
Thanks for pointing out. Now with new changes we will not need SessioManager to manage sessions. Will make necessary changes.